api/provider/aws/service_create.go

```diff
@@ -58,13 +82,22 @@ func (s *ServiceProvider) Create(req models.CreateServiceRequest) (string, error
 		scale = 1
 	}
 
+	privateSubnets := s.Config.PrivateSubnets()
```

Comment:
Rule of Thumb! Don't let aws-sdk-go's ugliness leak out of helper functions (when you can help it). The helper should take `[]string` instead of `[]*string`, and manage the ugly conversions within itself.

Reply:
I think keeping it as-is is fine - it's maybe not perfectly in-line with the current pattern, but doesn't seem like a big deal to me.